src/core/control/tools/StrokeStabilizer.cpp

```diff
@@ -128,10 +128,9 @@ void StrokeStabilizer::Active::quadraticSplineTo(const Event& ev) {
     Point sp = C.relativeLineTo(Q, 2.0 / 3.0);
 
     /**
-     * Set the pressure values. We assume the tool is pressure sensitive:
-     *      stroke->getToolType() == STROKE_TOOL_PEN
+     * Set the pressure values. Only the usual pen strokes are pressure sensitive
      */
-    bool usePressure = ev.pressure != Point::NO_PRESSURE;
+    bool usePressure = ev.pressure != Point::NO_PRESSURE && stroke->getToolType() == STROKE_TOOL_PEN;
```

Comment:
I think it would be better to add and use a method like `canToolTypeHavePressure(toolType)` instead of assuming that only pen is pressure-sensitive, in case we want to add another pressure-sensitive tool in the future.